src/Sylius/Component/Core/Pricing/AbstractCalculator.php

```diff
@@ -31,7 +41,11 @@ public function __construct()
     }
 
     /**
-     * {@inheritdoc}
```

Comment:
IMHO this class should implement `CalculatorInterface` and still have that inheritdoc.